packages/federation/src/gateway.ts

```diff
@@ -232,3 +233,75 @@ export const federationSubschemaTransformer: SubschemaConfigTransform =
       },
     };
   };
+
+export async function getStitchedSchemaFromManagedFederation(options: {
+  graphRef: string;
+  apiKey: string;
+  upLink?: string;
+  lastSeenId?: string;
+  abortSignal?: AbortSignal;
```

Comment:
```suggestion
  abortSignal?: AbortSignal;
```
We can accept a custom `fetch` directly here.